io/disk/softwareraid.py.data/Readme

```diff
@@ -2,22 +2,15 @@ mdadm - manage MD devices aka Linux Software RAID
 
 Values to be passed in yaml file:
 
-* Devices on which this scripts should be run. disks: "/dev/sdx /dev/sdy ..." (comman separated disks)
+* Devices on which this scripts should be run. disks: "/dev/sdx /dev/sdy ..." (comma separated disks)
 
 * Raid Name to be created, optional parameter.
 
 * Raid levels to be created
 
 * metadata levels for raid
 
-* Setup - True if only raid needs to be created and exit
-
-* Cleanup - True if only raid needs to be deleted and exit
-
-* Run_tests - True if only mdadm tests (like add, fail, remove device, etc)
-needs to be run on created raid
-
-* force - True if force option needs to be passed while creating raid.
+* spare disks (comma separated disks) [optional parameter]
```

Comment:
@narasimhan-v dude, this should be space separated not comma. can you please re-check this.

Reply:
sorry, my bad.